### PR TITLE
Commcalls

### PR DESCRIPTION
**Pull request by maelle**
> Here are some requests based on prepping the Dec 16 comm call announcement (I'll do that in a separate PR). 

@stefaniebutland here are some fixes or tries at fixes.

> You asked me to check R-bloggers R-Weekly appearance I think, but I don't know how to do that. 

It seems the comm call does end up in the RSS feed at rbloggers/index.xml

image

> [x] Can you please stop YAML `description:` from being displayed in the single comm call page? (see annotated image)

Done, now it only appears for comm calls that have no content (old comm calls, where the description is the only context we can offer website visitors).

> [x] I might want to use `###` headings summary of event, but these lovely link symbols appear when I do that 😉 (see annotated image)

Removed that effect from call calls.

> [ ] Should commcall [YAML instructions](https://github.com/ropensci/roweb3#comm-calls) be `title: Community Call - Call Title in Title Case`? 
 >  - Without "Community Call" in title, person won't easily recognize it as a comm call on [Events page](https://deploy-preview-73--ropensci.netlify.app/events/) 
 >  - Without "Community Call" in title, Twitter Card does not indicate it's a community call (see image)

I see your point but prefer not to make the title longer so

* I added "(Community Call)" for comm calls on the event page. Note that comm calls already appear with a different icon but it's not very obvious.
* I added "· Community Call" to og:title of community calls.

> - [ ] If you agree with changing YAML title instructions, then please remove "Comm Call " from the google cal (and iCal?) event name. Keep "rOpenSci" there, so event will appear in calendar as "rOpenSci Community Call - Call Title in Title Case" (see image)

Not needed.

> - [ ] In google cal (and iCal?) Location, please omit text "Online via Zoom". With only the URL there, a user can click the Location link and go directly to the meeting

I did that for Google calendar but the link does not seem clickable when I add it to my calendar, though.

> - [ ] Possible edits to "Join Us" section to be resolved in a ~15min chat
  - regarding zoom info, some people need direct link (that has passwd embedded) and some need meeting ID and passwd, and some folks do join by phone, so lots of info to share. Could that be in a separate text file that is unique for each call? Let's discuss first. 

I booked the chat but also added our old strategy of using details. Also note, in this PR, how I added all details via YAML. 

* We will indeed discuss how to make info appear.
* If part of the info never changes we can discuss where to store it if not in YAML.

### CODE REVIEW

File: content/events/2020-10-22-rladies-east-lansing.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: R-Ladies East Lansing Meetup
```
Comment (maelle) [inline content/events/2020-10-22-rladies-east-lansing.md:2]: Just so we might see a calendar with both an event and a comm call.

**Comment by maelle** [comment]
image

**Comment by maelle** [comment]
I've actually made an extreme proposal in this PR, by removing the "join us" section and making Zoom info a popover near the date. If you like the idea, it'd need to be polished, for instance I haven't been able to stop the sanitizing of the popover so the link isn't clickable which is not good.

**Comment by maelle** [comment]
Much improved collapsible Zoom info at the top, thanks @jeroen for the idea!!

File: content/events/2020-10-22-rladies-east-lansing.md (hunk 1 above)
Comment (stefaniebutland) [reply content/events/2020-10-22-rladies-east-lansing.md:2]: Events page preview shows only individual event now?
https://deploy-preview-78--ropensci.netlify.app/events/

Comment (maelle) [reply content/events/2020-10-22-rladies-east-lansing.md:2]: no I see both events?

**Pull request merged**